Jenkinsfile

```diff
@@ -61,15 +61,45 @@ spec:
                     unstash 'deployables'
                     sh '''#!/usr/bin/env bash
 
-                    export sshHost="genie.codewind@projects-storage.eclipse.org"
-                    export deployParentDir="/home/data/httpd/download.eclipse.org/codewind/codewind-vscode"
-
+                    export REPO_NAME="codewind-vscode"
+		    export OUTPUT_NAME="codewind"
+	       	    export OUTPUT_THEIA_NAME="codewind-theia"
+ 		    export OUTPUT_DIR="$WORKSPACE/dev/ant_build/artifacts"
```

Comment:
can we name this something other than `ant_build`

Reply:
actually it appears to not be used at all

Reply:
Good catch. Fixed.